Makefile.am

```diff
@@ -163,6 +163,17 @@ if BUILD_WINDOWS
 ecdsa_example_LDFLAGS += -lbcrypt
 endif
 TESTS += ecdsa_example
+if ENABLE_MODULE_RECOVERY
+noinst_PROGRAMS += recovery_example
+recovery_example_SOURCES = examples/recovery.c
+recovery_example_CPPFLAGS = -I$(top_srcdir)/include -DSECP256K1_STATIC
```

Comment:
I understand that this line follows the pattern, but `-DSECP256K1_STATIC` flag is relevant for Windows only:https://github.com/bitcoin-core/secp256k1/blob/20e3b4474639edf8fab6766ac6bda161bd9f52e5/include/secp256k1.h#L165-L168

Reply:
I see, so a separate PR could make sense where this is moved into the `if BUILD_WINDOWS` block for each example? Happy to open one (or review) if that is what you had in mind.